### PR TITLE
Admin guide: Topic rearrangements and formatting changes.

### PR DESCRIPTION
**Pull request by nbalacha**
Signed-off-by: N Balachandran <nbalacha@redhat.com>

### CODE REVIEW

File: Administrator Guide/index.md [hunk 1]
```diff
@@ -9,34 +9,38 @@
...
 
-4.  [Accessing Data - Setting Up Clients](./Setting Up Clients.md)
+	*  [Managing the Gluster Service](./Start Stop Daemon.md)
+	*  [Setting Up Trusted Storage Pools](./Storage Pools.md)
+       
+4. Setting Up Storage
+
+	*  [Brick Naming Conventions](./Brick Naming Conventions.md)
+	*  [Formatting and Mounting Bricks](./formatting-and-mounting-bricks.md)
+	*  [POSIX Access Control Lists](./Access Control Lists.md)
```
Comment (prashanthpai) [inline Administrator Guide/index.md]: Isn't this a client side operation ? IOW, it needs a mounted volume.

File: Administrator Guide/setting-up-storage.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Setting Up Storage
+
+A volume is a logical collection of bricks where each brick is an export directory on a server in the trusted storage pool.
+Before creating a volume, you need to set up the bricks that will form the volume.
+
+
+ -  [Brick Naming Conventions](./Brick Naming Conventions.md)
+ -  [Formatting and Mounting Bricks](./formatting-and-mounting-bricks.md)
+ -  [Posix ACLS](./Access Control Lists.md)
```
Comment (prashanthpai) [inline Administrator Guide/setting-up-storage.md]: Same comment here about ACLs.

Comment (nbalacha) [reply Administrator Guide/setting-up-storage.md]: The page actually talks abt mounting the brick with -o acl .

File: Administrator Guide/index.md (hunk 1 above)
Comment (nbalacha) [reply Administrator Guide/index.md]: The page actually talks abt mounting the brick with -o acl .

**Pull request merged**